submissions/kuslosolodka/html-css-popup/index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Popup</title>
+    <link rel="stylesheet" href="styles/style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="popup-button">
```

Comment:
Please provide the ability to open a popup with the keyboard. When the user clicks the `Tab` key focus goes to the browser controls instead of your button, also, when the last item is in focus and the user clicks the `Tab` key focus must go to the button `More` but in your case it goes to the browser controls. 
[Screencast from 27-08-22 09:53:44.webm](https://user-images.githubusercontent.com/16196199/187019099-1c6912fd-7f6c-4fb0-a8f2-f87cfbad3dd8.webm)